### PR TITLE
Release 0.35.0

### PR DESCRIPTION
**Pull request by zippolyte**
### Review checklist (to be filled by reviewers)

- [ ] Feature or bug fix MUST have appropriate tests (unit, integration, etc...)
- [ ] PR title must be written as a CHANGELOG entry [(see why)](/CONTRIBUTING.md#pull-request-title)
- [ ] Files changes must correspond to the primary purpose of the PR as described in the title (small unrelated changes should have their own PR)
- [ ] PR must have one `changelog/` label attached. If applicable it should have the `backward-incompatible` label attached.
- [ ] PR should not have `do-not-merge/` label attached.
- [ ] If Applicable, issue must have `kind/` and `severity/` labels attached at least.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 CHANGELOG
 =========
+## 0.35.0 / 2020-03-06
```
Comment (gzussa) [inline CHANGELOG.md]: Let's group `Added` together

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+
+* [Added] Add `Set` metric type to threadstats. See [#545](https://github.com/DataDog/datadogpy/pull/545).
+* [Added] Add enum for monitor types. See [#544](https://github.com/DataDog/datadogpy/pull/544).
+* [Added] Support DD_API_KEY environment variable in dogwrap. See [#543](https://github.com/DataDog/datadogpy/pull/543).
+* [Fixed] Remove illegal characters from tags. See [#517](https://github.com/DataDog/datadogpy/pull/517). Thanks [jirikuncar](https://github.com/jirikuncar).
+* [Fixed] Fix syntax error in dogwrap timeout handler and always collect output (#537). See [#538](https://github.com/DataDog/datadogpy/pull/538). Thanks [Matt343](https://github.com/Matt343).
```
Comment (nmuesch) [inline CHANGELOG.md]: Should we keep the issue number here?

File: CHANGELOG.md (hunk 1 above)
Comment (zippolyte) [reply CHANGELOG.md]: Sorting by issue type needs to be added to `ddev` then

File: CHANGELOG.md (hunk 2 above)
Comment (zippolyte) [reply CHANGELOG.md]: no

File: CHANGELOG.md (hunk 1 above)
Comment (zippolyte) [reply CHANGELOG.md]: Added here: https://github.com/DataDog/integrations-core/pull/5995

**Pull request merged**